package/slowroll.patch

```diff
@@ -0,0 +1,76 @@
+commit ca6232de1d35cc0613dae655a9349cd3c460a0cf
+Author: Bernhard M. Wiedemann <bwiedemann@suse.de>
+Date:   Mon May 13 16:37:42 2024 +0200
+
+    Use Slowroll repos
+
+diff --git a/control/control.xml b/control/control.xml
+index c550f23..3860444 100644
+--- a/control/control.xml
++++ b/control/control.xml
+@@ -126,53 +126,53 @@ textdomain="control"
+         <online_repos_preselected config:type="boolean">false</online_repos_preselected>
+ 
+         <!-- FATE #300898, List of external sources accesible during the installation time -->
+-        <external_sources_link>https://download.opensuse.org/YaST/Repos/openSUSE_Factory_Servers.xml</external_sources_link>
++        <external_sources_link>https://download.opensuse.org/YaST/Repos/openSUSE_Slowroll_Servers.xml</external_sources_link>
+ 
+         <dropped_packages/>
+         <extra_urls config:type="list">
+             <!-- Default update repository, bnc #381360 -->
+             <extra_url>
+-                <baseurl>http://download.opensuse.org/update/tumbleweed/</baseurl>
++                <baseurl>http://download.opensuse.org/update/slowroll/repo/oss/</baseurl>
+                 <alias>repo-update</alias>
+-                <name>openSUSE-Tumbleweed-Update</name>
++                <name>openSUSE-Slowroll-Update</name>
+                 <prod_dir>/</prod_dir>
+                 <enabled config:type="boolean">true</enabled>
+                 <autorefresh config:type="boolean">true</autorefresh>
+-                <priority config:type="integer">99</priority>
++                <priority config:type="integer">80</priority>
```

Comment:
Yes, because our base repo-oss is just a Tumbleweed snapshot and we want the option to override things without always needing higher version/revision numbers.